scripts/post-build.fsx

```diff
@@ -0,0 +1,50 @@
+open System.IO
+
+let rec DirectoryCopy (sourceDirName: string) (destDirName: string) (copySubDirs: bool) =
+    // Get the subdirectories for the specified directory.
+    let dir = DirectoryInfo sourceDirName
+    let dirs = dir.GetDirectories()
+
+    // If the destination directory doesn't exist, create it.
+    Directory.CreateDirectory destDirName |> ignore
+
+    // Get the files in the directory and copy them to the new location.
+    let files = dir.GetFiles()
+
+    for file in files do
+        file.CopyTo(Path.Combine(destDirName, file.Name), false)
+        |> ignore
+
+    // If copying subdirectories, copy them and their contents to new location.
+    if copySubDirs then
+        for subdir in dirs do
+            let tempPath = Path.Combine(destDirName, subdir.Name)
+            DirectoryCopy subdir.FullName tempPath copySubDirs
+
+let ReplaceInFile (fileName: string) (before: string) (after: string) =
+    let beforeContent = fileName |> File.ReadAllText
+
+    let afterContent = beforeContent.Replace(before, after)
+
+    File.WriteAllText(fileName, afterContent)
+
+let RemoveFoldersWithUnderscorePrefix (rootDir: string) =
+    let dir = DirectoryInfo rootDir
+    let dirs = dir.GetDirectories()
+
+    for subdir in dirs do
+        if subdir.Name.StartsWith("_") then
+            Directory.Delete(subdir.FullName, true)
+
+let args = fsi.CommandLineArgs |> Array.tail
```

Comment:
you can use `Seq.tail`, no need to specify `Array` here, AFAIU

Reply:
using array let us use .[x] syntex in the following lines

Reply:
actually, this reminds me that you should check for length of args first, and failwith if it's different than 2; after you do that change, you can use .First() and .Last(), from System.Linq, instead of .[0] and .[1]

Reply:
this scripts was supposed to get called only by CI so it wan't important

Reply:
well then rename the PR to not mention CI, what we will have is much more simplified build that can be also used by a new CI lane